docs/resources/obs_bucket.md

```diff
@@ -288,9 +288,12 @@ The `lifecycle_rule` object supports the following:
   automatically deleted. (documented below).
 * `noncurrent_version_transition` - (Optional, List) Specifies a period when noncurrent object versions are
   automatically transitioned to `WARM` or `COLD` storage class (documented below).
+* `abort_incomplete_multipart_upload` - (Optional, List) Specifies a period when fragments that have been last updated
+  are automatically deleted. (documented below).
+  **NOTE**: abort_incomplete_multipart_upload does not support filtering by tag.
```

Comment:
- The description for field `abort_incomplete_multipart_upload` is inaccurate.
- This line of description information is ambiguous. Seems like this line can be removed.